less/navigationBar.less

```diff
@@ -671,6 +671,7 @@
     content: ' ';
     position: absolute;
     background: #fff;
+    border: 1px solid @focusUrlbarOutline;
```

Comment:
This needs removing too.

cf: https://github.com/cezaraugusto/browser-laptop/commit/7b4ab6ffc7a93385427a9424e1a1a74d6dd695f9